R/duplicate_argument_linter.R

```diff
@@ -22,11 +24,8 @@ duplicate_argument_linter <- function(except = character()) {
     xml <- source_expression$full_xml_parsed_content
 
     calls <- xml2::xml_find_all(xml, xpath_call_with_args)
-
-    if (length(except)) {
```

Comment:
we still need the `length(except)` test right? user can still pass `except = character()`, perhaps by accident.

Reply:
Ah, true. When I opened the PR, I was creating the exception vector inside the function, so the length could never be zero, but in the current implementation, it can indeed be 0.

Fixed this.